snyk_feed.py

```diff
@@ -70,14 +73,22 @@ def fetch(self):
             res = requests.get(self.URL, headers=headers)
         except Exception as e:
             logger.error('Unable to get snyk feed. Reason: %r' % e)
-        logger.info("Snyk feed successfully fetched. Storing in S3.")
-        self.store_data(res.text)
+        logger.info("Snyk feed successfully fetched.")
+        logger.info("Triggering Snyk ingestion.")
+        logger.info(res.text)
+        snyk_json = json.loads(res.text)
+        SnykCveSync(snyk_json).run_snyk_sync()
```

Comment:
It will be better if we have a function that will compose all of this. It will pick up the feed and then call the sync process. Starting the sync process from the feed collector does not seem correct.

Reply:
noted

Reply:
function added